heartbeat/monitors/wrappers/monitorstate/tracker.go

```diff
@@ -120,12 +126,9 @@ func (t *Tracker) GetCurrentState(sf stdfields.StdMonitorFields, rc RetryConfig)
 		if rc.waitFn != nil {
 			sleepFor = rc.waitFn()
 		}
-		logp.L().Warnf("could not load last externally recorded state, will retry again in %d milliseconds: %v", sleepFor.Milliseconds(), err)
+		logp.L().Warnf("could not load previous monitor state, retrying in %d milliseconds: %v", sleepFor.Milliseconds(), err)
 		time.Sleep(sleepFor)
 	}
-	if err != nil {
-		logp.L().Warnf("could not load prior state from elasticsearch after %d attempts, will create new state for monitor: %s", i+1, sf.ID)
```

Comment:
It might be nice to retain a specific mention to `sf.ID` in the logs, it's not something we have to track very regularly but we've had cases before.

Reply:
Thanks. Done https://github.com/elastic/beats/pull/39621/commits/7f0c24d5379aaf4162b822f79c32d10a363aaf6c